services/web/server/src/simcore_service_webserver/computation_subscribe.py

```diff
@@ -87,16 +88,21 @@ async def progress_message_parser(app: web.Application, data: bytes) -> bool:
         return await _handle_computation_running_progress(app, rabbit_message)
 
     # NOTE: other types of progress are transient
+    is_type_message_node = type(rabbit_message) == ProgressRabbitMessageNode
```

Comment:
- `type` is a more "precise". 
- `insistence` is true also for parent classes.
In the future we will be adding more of these and I'd like to avoid issues.